### PR TITLE
polish the ui

### PR DESCRIPTION
**Pull request by kimitaWanjohi**
- Polish up the UI.

### CODE REVIEW

File: src/components/Book.jsx [hunk 1]
```diff
@@ -6,17 +6,20 @@ export default function Book({ book }) {
...
+        </span>
       </div>
       <button
         type="button"
         onClick={() => dispatch(delBook(book.id))}
-        className="bg-red-500 hover:bg-red-400 py-1 px-2 text-white rounded-lg shadow-sm"
+        className="bg-blue-500 hover:bg-blue-400 py-1 px-2 text-white rounded-lg shadow-sm"
       >
         Delete Book
       </button>
```
Comment (AmaduKamara) [inline src/components/Book.jsx]: - [ ] Let's try to stick to the project requirements as this is setting us up for real world projects in the future when we get a real job. Kindly style your app to have all sections and placeholders even if they're not doing anything in the app at the moment but for project guideline and requirement, let's work according to how we're required to style the app. 👍🏽
- [ ] Currently your app does not mact the original design given, hence, you are to match the original design in order for your project to be complete. Please work on it 👍🏽

**Your App**

Screenshot 2023-05-12 at 6 13 31 PM

**The Original Design**

Screenshot 2023-05-12 at 6 11 24 PM

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️ ♻️ ♻️

Hi @kimitaWanjohi,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✅ You used Gitflow correctly 👍🏽
✅ The Linter checks passed 👍🏽

## Required Changes ♻️
_Check the comments under the review._

- [ ] Kindly style your app to match the original design as required 👍🏽
- [ ] I also noticed that, when I navigate from one page to another, the books get duplicated as many times as I navigate. Kindly fix that 👍🏽

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/components/Navbar.jsx [hunk 2]
```diff
@@ -1,27 +1,29 @@
 import { useNavigate } from "react-router-dom";
```
Comment (MeqdamAlqudah) [inline src/components/Navbar.jsx:1]: - [ ] Please as the last reviewer said you should work on your code and make it match the design currently these are the things that are missing in your design : 
- the navigation it doen't match the design the background should be white and the height should look as in the design  
- the chapter name and the button should match the design , your code 
image
 the design 
image
- there is a line between the chapter info and the progress bar ]
- the distance between the chapter  info and the progress bar should match the design 
image
- the border of the input field shoudl match the design and the text inside it color should match the design 
image
- the background of the card section should match the design 
image

- the distance between each book card should match the design 
image

**Review by MeqdamAlqudah — CHANGES_REQUESTED** [review]
Hi @kimitaWanjohi , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

image

 ## Highlights
:+1: colors 
:+1: project functionality 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ghost — APPROVED** [review]
Hi @kimitaWanjohi , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

✅ Correct use of Gitflow
✅ No linter error
✅ App is designed as expected 👍 



## Optional suggestions

N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**